test/integration/smart_answer_flows/calculate_statutory_sick_pay_test.rb

```diff
@@ -521,13 +521,13 @@ class CalculateStatutorySickPayTest < ActiveSupport::TestCase
     end
 
     should "not allow dates before 2011" do
-      add_response Date.parse("2010-12-31")
+      add_response "2010-12-31"
```

Comment:
A bunch of the dates broke the tests since they had slashes between segments, which meant they were understand as separate responses. The other tweaks just made the surrounding code more consistent.

I've moved it to a separate commit.